shared/qcommon/q_platform.h

```diff
@@ -127,8 +127,15 @@ along with this program; if not, see <http://www.gnu.org/licenses/>.
 
 	#define PATH_SEP '/'
 
+	// [M] Define it, define it now! DEWIT.
 	#if !defined(ARCH_STRING)
-		#error ARCH_STRING should be defined by the build system
+		#if defined(__i386__)
+			#define ARCH_STRING "x86"
```

Comment:
Like the message said...`ARCH_STRING` should be defined by the build system. I've fixed this in 9b1fcbe

Reply:
This file is also out of scope for the rend2 code - any changes here should really be done upstream in the OpenJK project. (I don't consider rend2 to be an official part of the OpenJK project yet, hence all the work being done in my personal fork)

In any case, I've fixed the problem with `ARCH_STRING` not being defined so these changes are not needed now :)